nucypher/blockchain/eth/constants.py

```diff
@@ -10,6 +10,7 @@
 STAKING_ESCROW_STUB_CONTRACT_NAME = "StakingEscrowStub"
 ADJUDICATOR_CONTRACT_NAME = "Adjudicator"
 TACO_APPLICATION_CONTRACT_NAME = "TACoApplication"
+TACO_CHILD_APPLICATION_CONTRACT_NAME = "TACoChildApplication"
```

Comment:
This PR is looking good overall!  I do have one comment related to this line:  To avoid needing to change this line depending on whether or not a node is running on testnet vs mainnet, I assert that we always use this contract name in the registry, which is the historical convention anyways.

Reply:
👍 I agree with that assertion. We'll need to modify some logic when deploying to lynx as part of work for https://github.com/nucypher/nucypher-contracts/pull/119

Reply:
I suppose even for Lynx still will be `TACoChildApplication` abi, right?

Reply:
Yes, the ABI is the same, but we need some magic when generating the lynx registry since the contract names for root and child application contracts are different - https://github.com/derekpierre/nucypher-contracts/blob/lynx-deployment/contracts/contracts/testnet/LynxSet.sol.